tests/jballerina-unit-test/src/test/resources/test-src/query/join-clause.bal

```diff
@@ -0,0 +1,92 @@
+type DeptPerson record {|
```

Comment:
Let's add license header for test files as well. There was a discussion and decided to add license header for test files.

Reply:
Added in https://github.com/ballerina-platform/ballerina-lang/pull/27492/commits/03f53110add3cd9ced674ea1fb58213d849bb898